src/javascript/crypto/e2e/vrf/sha3shake256.js

```diff
@@ -0,0 +1,353 @@
+/**
+ * @license
+ * Copyright 2015 Yahoo Inc. All rights reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Extends js-sha3 to support SHAKE256, and outputs byte array
+ */
+
+// var e2e = {vrf:module.exports};
+goog.provide('e2e.vrf.sha3Shake256');
+
+goog.require('goog.asserts');
+
+/*
+ * Imported from js-sha3 v0.3.1
+ * https://github.com/emn178/js-sha3
+ *
+ * Copyright 2015, emn178@gmail.com
+ * 
+ * Licensed under the MIT license:
+ * http://www.opensource.org/licenses/MIT
```

Comment:
Thanks @gyehuda. 
@diracdeltas, can you suggest which convention/approach you prefer?